src/vm/docs/mod.rs

```diff
@@ -581,10 +581,13 @@ const AT_BLOCK: SpecialAPI = SpecialAPI {
     description: "The `at-block` function evaluates the expression `expr` _as if_ it were evaluated at the end of the
 block indicated by the _block-hash_ argument. The `expr` closure must be read-only.
 
+Note: The block identifying hash must be a hash returned by the `id-header-hash` block information
+property. This hash uniquely identifies Stacks blocks, whereas `header-hash` is not necessarily unique.
```

Comment:
Might want to clarify that `id-header-hash` is globally unique across all Stacks forks, since it's the hash of both the Stacks block and the burnchain block that selected it.  The `header-hash` value is unique in the context of a single fork.

Reply:
Sure